dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/SymbolProblem.java

```diff
@@ -76,20 +76,20 @@ public boolean equals(Object other) {
     SymbolProblem that = (SymbolProblem) other;
     return errorType == that.errorType
         && symbol.equals(that.symbol)
-        && Objects.equals(targetClass, that.targetClass);
+        && Objects.equals(containingClass, that.containingClass);
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(errorType, symbol, targetClass);
+    return Objects.hash(errorType, symbol, containingClass);
   }
 
   @Override
   public String toString() {
     return MoreObjects.toStringHelper(this)
         .add("errorType", errorType)
         .add("symbol", symbol)
-        .add("targetClass", targetClass)
+        .add("containingClass", containingClass)
```

Comment:
Maybe also call `.omitNullValues()`?

Reply:
Added the method. Good to know that. Thanks.